pydash/functions.py

```diff
@@ -210,6 +211,32 @@ def callback(item):
         return pyd.some(obj, callback)
 
 
+def flip(func):
+    """Creates a function that invokes the method with arguments reversed.
+
+    Args:
+        func (function): Function to flip arguments for.
+
+    Returns:
+        function: Returns the new flipped function.
+
+    Example:
+
+        >>> flipped = flip(lambda *args: args)
+        >>> flipped(1, 2, 3, 4)
+        (4, 3, 2, 1)
+        >>> flipped = flip(lambda *args: [i * 2 for i in args])
+        >>> flipped(1, 2, 3, 4)
+        [8, 6, 4, 2]
+
+    .. versionadded:: TODO
+    """
+    def _flip(*args, **kargs):
```

Comment:
Per my previous comment, let's implement this as a class like the other functions in this submodule.